go.mod

```diff
@@ -2,6 +2,10 @@ module github.com/open-policy-agent/gatekeeper
 
 go 1.16
 
+replace golang.org/x/crypto => golang.org/x/crypto v0.0.0-20211202192323-5770296d904e // CVE-2021-43565
```

Comment:
Do we have a mechanism for removing these once the replace is no longer necessary so we don't get stuck on an old version of these libraries?

Reply:
NVM, I see this is a backport, so we're not likely to be upgrading dependencies. This LGTM if sozercan/chewong approve